packages/labs/ssr-dom-shim/package.json

```diff
@@ -20,6 +20,10 @@
     ".": {
       "types": "./index.d.ts",
       "default": "./index.js"
+    },
+    "./css-hook.js": {
```

Comment:
Just a heads up that there is an option to set an export condition
https://nodejs.org/api/module.html#enabling
> Where some-package has an ["exports"](https://nodejs.org/api/packages.html#exports) field defining the `/register` export to map to a file that calls `register()`, like the following _register-hooks.js_ example.

edit: though maybe this is what you indicated would be the follow up PR.

Reply:
> edit: though maybe this is what you indicated would be the follow up PR?

Yes, basically. But to be able to use it from `@lit-labs/ssr` I already need a specific exports entry in `@lit-labs/ssr-dom-shim`. This also allows it to be used directly from this package, if so desired.